src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java

```diff
@@ -47,14 +48,12 @@
                     45,                      // minimum match length
                     1000,                    // maximum coverage on breakpoint interval
                     15,                      // minimum number of evidentiary reads in called cluster
-                    20000000,                // guess for total kmers in BAM partition
-                    2,                       // minimum kmer count within partition of high frequency kmer
-                    250000,                  // guess for number of unique error-free kmers in one partition
-                    200,                     // high frequency kmer count
                     3,                       // maximum number of intervals a localizing kmer can appear in
                     3,                       // KmerCleaner min kmer count
                     125,                     // KmerCleaner max kmer count
                     600000,                  // KmerCleaner guess for number of unique error-free kmers per partition
+                    500,                     // maximum template names for an assembly kmer
+                    250000,                  // unique, error-free, scrubbed kmers per assembly partition
```

Comment:
Is this a guess for capacity/partitioning's sake, or a maximum or some other threshold? Can you make the comment more clear, and the argument doc below?